cdap-security/src/main/java/co/cask/cdap/security/authorization/AbstractAuthorizationService.java

```diff
@@ -210,9 +210,9 @@ private void updatePrivileges(Principal principal) throws Exception {
   private void validateCacheConfig() {
     if (!isAuthCacheEnabled()) {
       if (cacheEnabled) {
-        LOG.warn("Authorization policy caching is enabled ({} is set to true), however, this setting will have no " +
+        LOG.trace("Authorization policy caching is enabled ({} is set to true), however, this setting will have no " +
```

Comment:
does this get logged a lot? I think in production this message makes sense as a warning because the config is inconsistent. This should only be called once? In that case it is not a big deal to keep the warning